INFO  2021-12-16 17:48:05 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 17:48:05 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-16 18:06:02 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 18:06:02 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-16 18:14:07 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 18:14:07 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-16 19:22:57 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 19:22:57 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-16 19:25:02 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 19:25:03 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-16 19:51:25 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-16 19:51:25 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:25) - Сервер подключен к базе данных
INFO  2021-12-18 10:42:25 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-18 10:42:26 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:26) - Сервер подключен к базе данных
INFO  2021-12-19 12:38:24 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:55) - Сервер запущен на порту 8190
INFO  2021-12-19 12:38:25 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:26) - Сервер подключен к базе данных
INFO  2021-12-19 16:33:06 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:58) - Сервер упал
ERROR 2021-12-19 16:33:06 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:59) - Throwing
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:550) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:249) ~[?:?]
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at java.lang.Thread.run(Thread.java:832) ~[?:?]
INFO  2021-12-19 16:33:48 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:58) - Сервер упал
ERROR 2021-12-19 16:33:48 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:59) - Throwing
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:550) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:249) ~[?:?]
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at java.lang.Thread.run(Thread.java:832) ~[?:?]
INFO  2021-12-19 16:38:15 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:58) - Сервер упал
ERROR 2021-12-19 16:38:15 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:59) - Throwing
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:550) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:249) ~[?:?]
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.70.Final.jar:4.1.70.Final]
	at java.lang.Thread.run(Thread.java:832) ~[?:?]
INFO  2021-12-19 16:43:36 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:54) - Сервер запущен на порту 8190
INFO  2021-12-19 16:43:36 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:26) - Сервер подключен к базе данных
INFO  2021-12-19 16:48:35 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:54) - Сервер запущен на порту 8190
INFO  2021-12-19 16:48:35 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:26) - Сервер подключен к базе данных
INFO  2021-12-19 18:23:15 [main] ru.aberezhnoy.server.core.NettyServerService (NettyServerService.java:54) - Сервер запущен на порту 8190
INFO  2021-12-19 18:23:15 [main] ru.aberezhnoy.server.service.impl.databaseConnection.DatabaseConnectionServiceImpl (DatabaseConnectionServiceImpl.java:26) - Сервер подключен к базе данных
